components/brave_rewards/browser/rewards_service_impl.cc

```diff
@@ -1568,51 +1534,44 @@ void RewardsServiceImpl::SetConfirmationsIsReady(const bool is_ready) {
 
 void RewardsServiceImpl::ConfirmationsTransactionHistoryDidChange() {
     for (auto& observer : observers_)
-    observer.OnConfirmationsHistoryChanged(this);
+    observer.OnTransactionHistoryForThisCycleChanged(this);
 }
 
-void RewardsServiceImpl::GetConfirmationsHistory(
-    brave_rewards::ConfirmationsHistoryCallback callback) {
+void RewardsServiceImpl::GetTransactionHistoryForThisCycle(
+    brave_rewards::TransactionHistoryForThisCycleCallback callback) {
   if (!Connected()) {
     return;
   }
 
-  auto earnings_range = GetEarningsRange();
-
-  bat_ledger_->GetConfirmationsHistory(
-      earnings_range.first,
-      earnings_range.second,
-      base::BindOnce(&RewardsServiceImpl::OnGetConfirmationsHistory,
-                     AsWeakPtr(),
-                     std::move(callback)));
+  bat_ledger_->GetTransactionHistoryForThisCycle(
+      base::BindOnce(&RewardsServiceImpl::OnGetTransactionHistoryForThisCycle,
+          AsWeakPtr(), std::move(callback)));
 }
 
-void RewardsServiceImpl::OnGetConfirmationsHistory(
-    brave_rewards::ConfirmationsHistoryCallback callback,
+void RewardsServiceImpl::OnGetTransactionHistoryForThisCycle(
+    brave_rewards::TransactionHistoryForThisCycleCallback callback,
     const std::string& transactions) {
-  std::unique_ptr<ledger::TransactionsInfo> info;
-  if (!transactions.empty()) {
-    info.reset(new ledger::TransactionsInfo());
-    info->FromJson(transactions);
-  }
-
-  if (!info) {
+  if (transactions.empty()) {
     callback.Run(0, 0.0);
+    return;
   }
 
+  ledger::TransactionsInfo info;
+  info.FromJson(transactions);
+
```

Comment:
This is the worst change I've ever seen, whitespace is for idiots! - this comment is just a test :)